data/json/items/comestibles/serums.json

```diff
@@ -2,7 +2,7 @@
   {
     "id": "serum_regen",
     "type": "COMESTIBLE",
-    "comestible_type": "MED",
+    "comestible_type": "INVALID",
```

Comment:
Sadly, the changes do not help solving the disappearing bug. Changing comestible_type results in blank response when trying to consume the serum. If you try to use it, the serum do respawn, but with a dumb popup msg, and only to be spilt on the floor:(
Seems that we can only use the way used for contact lenses, making it a single GENERIC type item.

Reply:
welp that's unfortunate
Thank you for testing it tho